vendor.conf

```diff
@@ -0,0 +1,37 @@
+github.com/Microsoft/go-winio c40bf24f405ab3cc8e1383542d474e813332de6d
```

Comment:
They're not alphabetical, vndr doesn't care. I listed them in the same order Godep was listing them, because I used the Godeps.json file as a basis for this one.